tests/runtime/ray/test_integration.py

```diff
@@ -34,10 +38,15 @@
 def runtime(
     shared_ray_conn, tmp_path_factory: pytest.TempPathFactory, change_db_location
 ):
+    # We need to set this env variable to let our logging code know the
+    # tmp location has changed.
+    os.environ[RAY_TEMP_PATH_ENV] = str(shared_ray_conn._temp_dir)
```

Comment:
Went for option b since `monkeypatch` is function-scoped [2cf8f9e](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/301/commits/2cf8f9e4951450f143155e2d6366ba88371300ee)